src/libs/runtime-utils.js

```diff
@@ -127,9 +128,9 @@ const dataprocCost = (machineType, numInstances) => {
 }
 
 const getHourlyCostForMachineType = (machineTypeName, region, isPreemptible) => {
-  const { cpu, memory } = _.find({ name: machineTypeName }, machineTypes)
+  const { cpu, memory } = _.find({ name: machineTypeName }, machineTypes) || {}
```

Comment:
I didn't worry too much about cost in this PR, as there's a future ticket to ensure it displays properly. https://broadworkbench.atlassian.net/browse/IA-3348

The title wording is a recent change, if you open the modal by selecting the tool in the context bar on the right, it will display the tool name in the title. Equivalent for Jupyter shown below. 

![image](https://user-images.githubusercontent.com/6465084/166304023-e4481676-df28-4001-9b89-dff0f0a9a033.png)